beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java

```diff
@@ -221,7 +221,7 @@ public void reload() {
             })
         .reduce((left, right) -> Union.of(left, right).output())
         .orElseThrow(failNotFound(attrs, "commit-log"))
-        .apply(filterAttrs(attrs));
+        .apply(String.format("get-stream-%s-filter-attrs-%s", name, attrs), filterAttrs(attrs));
```

Comment:
Can we remove this and the two following names for filter? That should be unnecessary. The rest can be merged. :+1:

Reply:
reverted 👍